sdks/go/pkg/beam/runners/prism/internal/unimplemented_test.go

```diff
@@ -52,7 +52,6 @@ func TestUnimplemented(t *testing.T) {
 		{pipeline: primitives.TriggerAfterProcessingTime},
 		{pipeline: primitives.TriggerAfterSynchronizedProcessingTime},
 		{pipeline: primitives.TriggerElementCount},
-		{pipeline: primitives.TriggerNever},
```

Comment:
Since this is default, does it make sense to remove this?

Reply:
Tests removed from this list *must* be added to another test that expects them to pass. Otherwise we're opening a test regression gap for ourselves down the road.

So, if you're removing this because TestUnimplmented started to fail, then you *must* move it to the TestImplmenented list down below.

If this test *didn't* start to fail, then we should leave it here to show we aren't fully done work on TriggerNever.